hepstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -3,7 +3,7 @@
 from scipy.stats import norm
 
 from .basecalculator import BaseCalculator
-from ..fitutils.utils import eval_pdf, array2dataset, pll
+from ...utils.fit import eval_pdf, array2dataset, pll
```

Comment:
Apologies @marinang for only being able to look properly at this new addition of the package today.

There is a dot too many in this line.

Reply:
I guess you will create a new PR now that this one have been merged ...

Reply:
Yeah I can do that.

You would prefer me doing:

```python
from hepstats.utils.fit import eval_pdf, array2dataset, pll
```
?

Reply:
You can still keep a relative import. If you import these `eval_pdf` and others often then I would make them available in the `utils/__init__.py` file … No need to have them "hidden" and require an explicit import from the actual `fit.py` file a level down.